handoffs/templates/planner.md

```
@@ -0,0 +1,155 @@
+- [ ] Check 3: <Likely findings check>
+
+# Dispatch comment draft
+
+```markdown
+Planned #N in `handoffs/issue-N-implementation-plan.md` on `main`.
+
+- **Implementer tier:** <Model Tier>
+- **Implementation branch:** `<allocated-branch-name>`
+- **Likely findings:** <Brief summary of predicted review focus areas>
+```
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)